packages/instant/src/components/zero_ex_instant.tsx

```diff
@@ -21,28 +25,34 @@ export type ZeroExInstantProps = ZeroExInstantRequiredProps & Partial<ZeroExInst
 export interface ZeroExInstantRequiredProps {
     // TODO: Change API when we allow the selection of different assetDatas
     assetData: string;
-    // TODO: Allow for a function that returns orders
-    liquiditySource: string;
+    liquiditySource: string | SignedOrder[];
 }
 
 export interface ZeroExInstantOptionalProps {
+    defaultAssetBuyAmount?: number;
     additionalAssetMetaDataMap: ObjectMap<AssetMetaData>;
     network: Network;
 }
 
 export class ZeroExInstant extends React.Component<ZeroExInstantProps> {
     private readonly _store: Store;
     private static _mergeInitialStateWithProps(props: ZeroExInstantProps, state: State = INITIAL_STATE): State {
-        // Create merged object such that properties in props override default settings
-        const optionalPropsWithDefaults: ZeroExInstantOptionalProps = {
-            additionalAssetMetaDataMap: props.additionalAssetMetaDataMap || {},
-            network: props.network || state.network,
-        };
-        const { network } = optionalPropsWithDefaults;
+        const network = props.network || state.network;
```

Comment:
on another note, we probably want to add asserstions for the props since this wil be a public facing API, perhaps in the `render` method in `index.umd.ts` ?